go.mod

```diff
@@ -6,80 +6,85 @@ require (
 	github.com/DATA-DOG/go-sqlmock v1.5.0
 	github.com/DATA-DOG/go-txdb v0.1.3
 	github.com/DataDog/datadog-go v4.8.2+incompatible
-	github.com/aws/aws-sdk-go v1.34.28
+	github.com/aws/aws-sdk-go v1.44.327
 	github.com/getsentry/sentry-go v0.12.0
 	github.com/go-kit/kit v0.9.0
 	github.com/google/uuid v1.6.0
 	github.com/grpc-ecosystem/go-grpc-middleware v1.0.0
 	github.com/magefile/mage v1.15.0
+	github.com/redis/go-redis/v9 v9.5.1
```

Comment:
@fotos Yeah, I have committed both adding a Redis new library and upgrading the `dd-trace-go` in one commit. Don't think it makes sense to separate them since the redis library is a dependency for the `dd-trace-go` anyway